ang/civicase/activity/actions/services/move-copy-activity-action.service.js

```diff
@@ -63,6 +65,20 @@
       });
     }
 
+    /**
+     * @returns {object} api parameters for Case.getlist
+     */
+    function getCaseListApiParams () {
+      return {
+        'case_type_id.case_type_category': _.keys(CaseTypeCategory.getCategoriesWithAccessToBasicCaseInformation())[0],
```

Comment:
let's move this part to a variable, it's too long:

```js
var firstCaseTypeCategoryId =
  _.keys(CaseTypeCategory.getCategoriesWithAccessToBasicCaseInformation())[0];
var allCaseTypeCategoryNames =
  _.values(CaseTypeCategory.getCategoriesWithAccessToBasicCaseInformation());

return {
  'case_type_id.case_type_category': firstCaseTypeCategoryId,
  params: {
    'case_id.case_type_id.case_type_category': {
      IN: allCaseTypeCategoryNames
    }
  }
}
```

Reply:
Also, a big question. I know it was in the description, but why do we send the first case category ID plus the full list of case category names? Can you please explain a bit more about that?